rtl/include/eclhelper_base.hpp

```diff
@@ -3721,6 +3721,33 @@ class CLibraryExistsAggregateArg : public CThorExistsAggregateArg
     IOutputMetaData * meta;
 };
 
+class CLibraryConstantRawIteratorArg : public CThorLinkedRawIteratorArg
+{
+public:
+    inline CLibraryConstantRawIteratorArg(unsigned _numRows, byte * * _rows, IOutputMetaData * _meta)
+        : meta(_meta), numRows(_numRows), rows(_rows)
+    {
+        cur = 0;
+    }
+
+    virtual void onStart(const byte *, MemoryBuffer * ) { cur = 0U; }
```

Comment:
Is it within the boundaries of our code practices to leave these parameters nameless?